apps/condo/domains/common/hooks/useSearchByPhoneModal.tsx

```diff
@@ -224,18 +217,21 @@ export const useSearchByPhoneModal = (searchByPhoneFn, canManageContacts) => {
     const intl = useIntl()
     const SearchByPhoneMessage = intl.formatMessage({ id: 'SearchByPhoneNumber' })
 
+    const { isSmall } = useLayoutContext()
+
     const [isSearchByPhoneModalVisible, setIsSearchByPhoneModalVisible] = useState<boolean>(false)
 
     const handleCloseModal = useCallback(() => setIsSearchByPhoneModalVisible(false), [])
 
     const SearchByPhoneModal = useMemo(() => (
         <StyledModal
-            visible={isSearchByPhoneModalVisible}
+            isSmall={isSmall}
+            open={isSearchByPhoneModalVisible}
             title={SearchByPhoneMessage}
             onCancel={handleCloseModal}
             footer={null}
-            width={1150}
             destroyOnClose
+            centered={false}
```

Comment:
You should not destroy our component just because you need a custom in on place. All our modals are centered by default then used as a real modals....